configure.ac

```diff
@@ -541,6 +544,12 @@ AC_SUBST(LIBSTDCXX_LA_LINKER_FLAG)
 # In fact, a lot of the code in this directory depends on pthreads
 AX_PTHREAD
 
+AC_LINK_IFELSE([AC_LANG_PROGRAM([#include <sys/types.h>],
```

Comment:
why? Can we simply do "windows doesn't have it, everyone else has" ? Because in practice it'll more or less like that.

Reply:
Unfortunately MinGW may have `pid_t`. And it seems to be inconsistent between flavours of MinGW.